odbc-api/src/handles.rs

```diff
@@ -24,12 +25,14 @@ pub use {
     column_description::{ColumnDescription, Nullability},
     connection::Connection,
     data_type::DataType,
-    diagnostics::{State, Record},
+    diagnostics::{Record, State},
     environment::Environment,
     error::Error,
     statement::{ParameterDescription, Statement, StatementImpl},
 };
 
+pub(crate) use columns::columns;
```

Comment:
I wasn't sure if we'd prefer to organize this differently somehow. I was trying to find some other examples of shared helpers, but some of them had a natural placement (e.g. they didn't need to access internals from other modules)